site/themes/hugo-theme-relearn/layouts/_default/_markup/render-link.html

```diff
@@ -4,19 +4,20 @@
 {{ $fragment := "" -}}
 {{ with $url.Fragment }}{{ $fragment = printf "#%s" . }}{{ end -}}
 {{ $permalink := "" }}
-
-{{ $isRemote := ne $url.Scheme "" -}}
```

Comment:
Does this not work? The advantage is that it catches any scheme, `http`, `https`, `ftp`, etc.
We do have links to both HTTP and HTTPS.